### PR TITLE
Use npm scripts rather than Makefile

### PR DESCRIPTION
**Pull request by dgaubert**
Makefile was something needed in the past because of the lack of tools in the early days of Node.js. Nowadays nobody uses Makefile for a Node.js project because it comes with built-in solutions (npm scripts for this use-case).

### CODE REVIEW

File: test/setup.sh [hunk 1]
```diff
@@ -94,25 +85,14 @@ die() {
...
 trap 'die' HUP INT QUIT ABRT TERM;
 
-
-
 # Database setup
 cleanup;
 create_db;
 
-# echo -e "\nInstalling cartodb extension";
+echo -e "\nInstalling cartodb extension";
```
Comment (Algunenano) [inline test/setup.sh]: Please delete this command, it was used before calling `CREATE EXTENSION cartodb` but that was removed before.

**Comment by dgaubert** [comment]
It was on purpose, what do we need to clean-up after the tests are run? Am I missing something?

In CIs the instance is going to disappear anyway. Whereas in your localhost you might need to check the status of the DB if something went wrong while testing. You only need to get a clean DB before running the test again.

**Comment by Algunenano** [comment]
> It was on purpose, what do we need to clean-up after the tests are run? Am I missing something?

Because it leaves trash in the cluster and that's a pain in the ass to later maintain, update and everything else. Tests should not leave trash outside of their controlled environment. `Boy Scout Rule` applied to the environment, you should leave the database cleaner than you found it.

> In CIs the instance is going to disappear anyway. Whereas in your localhost you might need to check the status of the DB if something went wrong while testing. You only need to get a clean DB before running the test again.

The makefile has a flag to keep the database if you want it. The new system should be as flexible as good as the previous one or we are going backwards.

**Comment by Algunenano** [comment]
How do I run the test with thw option?

`VERBOSE=yes npm test` doesn't work.
`npm test --verbose` does a different thing and outputs npm stuff.

**Comment by dgaubert** [comment]
`"test:setup": "./test/setup.sh --verbose"`

**Pull request merged**